spec/fixtures/test_module_rpms/pupmod-simp-beakertest-0.0.1/Gemfile.lock

```diff
@@ -1,463 +1,270 @@
-GIT
-  remote: https://github.com/trevor-vaughan/rubygem-simp-rake-helpers
-  revision: 34e01602fe294d294e74caefd060960e23bed0c5
```

Comment:
This file should not be removed.  It is a historical record of the (buggy) version of simp-rake-helper used to create this RPM.